### DISCUSSION TITLE
zerotier

### DISCUSSION THREAD

**Discussion by ccsshcn** [immortalwrt/immortalwrt#1189] [General]
### Describe the bug

R2S用这里的固件安装zerotier后，无法被远程操控。用iStoreOS的固件安装zerotier后，能够被远程操控。比较了一下已安装的软件包，发现这里的固件少了app-meta-zerotier

### ImmortalWrt version

ImmortalWrt 23.05.1

### ImmortalWrt release

r27304-31bc47589e

### ImmortalWrt target/subtarget

zerotier

### Device

FriendlyElec NanoPi R2S

### Image kind

Self-built image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by liyiranlab** [comment]
感觉 贵州移动 已经成功屏蔽zerotier
